config/deploy.secret.exs

```diff
@@ -0,0 +1,10 @@
+use Mix.Config
+
+# Configure your database
```

Comment:
This is ok for this build, but consider the following:
- mix compile will be using the ENV on the machine that's doing the compiling... so the environment becomes "compiled into" the binary.
- it's important to note that compilation for another platform or deployment needs prod.secret.exs to be redefined for every deployment.